### ISSUE TITLE
add description paramater to manager.create_device

### ISSUE THREAD

**Issue by pnhowe** [packethost/packet-python#121]
please add the "description" parameter to the create_device method of the Manager.  Also I noticed a few other missing parameters: ssh_keys, no_ssh_keys, private_ipv4_subnet_size, ip_addresses.  Might be nice to have all the API's create device parameters exposed.

**Comment by displague** [comment]
Indeed @pnhowe.  This client is behind API parity in a number of ways.  I'm flagging these attributes to be added in a near release, but I don't have any specific timelines.

The Equinix Metal API Swagger2/OAS2 Spec is public and can be used to generate a client, as in https://github.com/displague/metal-python/blob/main/metal/types/device_create_input.py

I'm working towards making generated clients such as this the primary go-to client and eventually deprecating handwritten clients such as this.

Let us know if you are blocked by this or if you would like to have any contributions reviewed and released to add this functionality.

https://github.com/packethost/packet-python/pull/91 is an open PR to add `ip_address` and we could further that along if it helps.

**Comment by displague** [comment]
`created_by` is an attribute that is missing from the Device class which some users have expressed a desire to have access to.
